src/test/java/de/wejago/mqtt2influx/service/JsonSubscriberTest.java

```diff
@@ -0,0 +1,88 @@
+package de.wejago.mqtt2influx.service;
+
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.wejago.mqtt2influx.config.Device;
+import de.wejago.mqtt2influx.repository.InfluxDbRepository;
+import java.util.HashMap;
+import java.util.Map;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class JsonSubscriberTest {
+    @Mock
+    private ObjectMapper objectMapper;
+    @Mock
+    private InfluxDbRepository influxDbRepository;
+    @Mock
+    private Device device;
+
+    @InjectMocks
+    private JsonSubscriber jsonSubscriber;
+
+    @Test
+    void messageArrived_whenOnlyMatchFound() throws Exception {
+        // GIVEN
+        MqttMessage mqttMessage = new MqttMessage("testPayload".getBytes());
+        String json = "{\"Time\":\"2023-05-10T10:16:32\",\"\":{\"Total_in\":695.38,\"Total_out\":16.94,\"Power_curr\":1151," +
+                      "\"device_id\":\"0a01454d480000b22b25\"}}";
+        ObjectMapper realObjectMapper = new ObjectMapper();
+        JsonNode jsonNode = realObjectMapper.readTree(json);
+        JsonNode jsonNodeMapping = jsonNode.get("");
+        Map<String, String> deviceMappings = new HashMap<>();
```

Comment:
Create the device configuration in a method and get it here, that makes the test cleaner.